File: test/fixtures/python-sample/src/api/routes/health.py
"""Health check routes module."""

from datetime import datetime
from fastapi import APIRouter

router = APIRouter()


@router.get("/")
async def health_check() -> dict:
    """Health check endpoint."""
    return {
        "status": "healthy",
        "timestamp": datetime.now().isoformat(),
    }


@router.get("/ready")
async def readiness_check() -> dict:
    """Readiness check endpoint."""
    # TODO: Check database connection
    return {"ready": True}

File: test/fixtures/typescript-sample/src/routes/health.ts
import { Router, Request, Response } from 'express';

const router = Router();

router.get('/', (req: Request, res: Response) => {
  res.json({
    status: 'healthy',
    timestamp: new Date().toISOString(),
    uptime: process.uptime(),
  });
});

router.get('/ready', (req: Request, res: Response) => {
  // Check database connection, external services, etc.
  res.json({ ready: true });
});

export { router as healthRouter };
